revise about page (#325)
jspecify/jspecify: docs/src/pages/about.md

| [SpotBugs](http://spotbugs.rtfd.io/)                    | [SpotBugs Team](https://github.com/spotbugs/) |
| (various)                                               | [Square](https://squareup.com)                |

## What are we trying to do?

Java static analysis tools, such as Error Prone and those built into IDEs, have been very successful
at enabling automated refactorings and bug prevention.

Many of their most beneficial analyses require certain Java annotations (such as `@Nullable` or

> `@Immutable`) to be present in the source or class file. Yet for the going-on-two-decades since 

Reviewer: nit: "source or class file being analyzed" or pluralize it: "in source or class files"

Author: done

| [SpotBugs](http://spotbugs.rtfd.io/)                    | [SpotBugs Team](https://github.com/spotbugs/) |
| (various)                                               | [Square](https://squareup.com)                |

## What are we trying to do?

Java static analysis tools, such as Error Prone and those built into IDEs, have been very successful
at enabling automated refactorings and bug prevention.

Many of their most beneficial analyses require certain Java annotations (such as `@Nullable` or
`@Immutable`) to be present in the source or class file being analyzed. Yet for the 
going-on-two-decades since Java annotations were introduced, there's been no serious and 
successful effort to provide a single well-specified standard for these annotations with 
industry consensus. Instead, each tool has generally provided its own, and each tool interprets 
the meanings of the annotations differently.

> We think the lack of such an artifact has been one of the factors constraining the prevalance and 

Reviewer 2: ```suggestion
We think the lack of such a standard has been one of the factors constraining the prevalance and 
```

You haven't said the word "artifact" yet.

Author: fixed in the other direction.
